fhir-operation-validate/src/main/java/com/ibm/fhir/operation/validate/ValidateOperation.java

```diff
@@ -78,12 +78,16 @@ private OperationOutcome buildResourceValidOperationOutcome(List<Issue> issues)
                             .build())
                         .build());
         }
+        
+        boolean hasError = issues.stream().anyMatch(issue -> IssueSeverity.ERROR.equals(issue.getSeverity())
+                || IssueSeverity.FATAL.equals(issue.getSeverity()));
                 
         OperationOutcome operationOutcome = OperationOutcome.builder()
-                .id(Id.of("NoError"))
+                .id(Id.of(hasError? "Error" : "NoError"))
                 .text(Narrative.builder()
                     .status(NarrativeStatus.ADDITIONAL)
-                    .div(xhtml("<div xmlns=\"http://www.w3.org/1999/xhtml\"><p>No ERROR</p></div>"))
+                    .div(xhtml(hasError? "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p>ERROR</p></div>"
+                            : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p>NoERROR</p></div>" ))
```

Comment:
why no space?  this is the human-readable description and should have spaces between words.

Reply:
```suggestion
                            : <div xmlns=\"http://www.w3.org/1999/xhtml\"><p>No ERROR</p></div>" ))
```

(don't forget DCO if you accept this)